doc/hacking.rst

```diff
@@ -34,3 +34,26 @@ locally:
 * migrate database:  ./manage.py migrate
 * create superuser:  ./manage.py createsuperuser
 * anonymize data:    ./manage.py prepdump # avoid mailing users
+
+
+Running Javascript unit tests
+-----------------------------
+
+In order to run Javascript unit tests, you need to setup an npm package and
+install the dependencies first from the package-lock.json file.
+Here's the instruction of how it's done on Debian from squad base directory::
+
+  sudo apt-get install software-properties-common chromium
+  curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -
+  sudo apt-get install nodejs
+  npm install
+  sudo npm install -g karma-cli
+  export CHROME_BIN=/usr/bin/chromium
```

Comment:
maybe the (python) test file could detect this so we don't need to do this on every new shell